examples/homepage/install/x86_64/amplify.txt

```diff
@@ -6,7 +6,7 @@
 ? Do you want to configure advanced settings? Yes
 ...
 ? Do you want to enable Lambda layers for this function? Yes
-? Enter up to 5 existing Lambda layer ARNs (comma-separated): arn:aws:lambda:eu-central-1:017000801446:layer:AWSLambdaPowertoolsPythonV2:69
+? Enter up to 5 existing Lambda layer ARNs (comma-separated): arn:aws:lambda:eu-central-1:017000801446:layer:AWSLambdaPowertoolsPythonV3-312:69
```

Comment:
```suggestion
? Enter up to 5 existing Lambda layer ARNs (comma-separated): arn:aws:lambda:eu-central-1:017000801446:layer:AWSLambdaPowertoolsPythonV3-312:1
```

question: version 1 like others?

Reply:
I had to update this with the new name format we decided on. It's up to date.